Homeworks/Homework1/H1Task1/H1Task1/Sorting.cs

```diff
@@ -0,0 +1,21 @@
+namespace H1Task1
+{
+    public static class InsertionSort
+    {
+        /// <summary>
+        /// Sorts the elements in the list of ints
+        /// </summary>
+        public static IEnumerable<int> Sort(IList<int> listToSort)
```

Comment:
```suggestion
        public static IEnumerable<int> Sort(List<int> listToSort)
```
В рамках поставленной задачи (на вход стандартный массив) всё ок, сортировка квадратична.
Но в общем случае для такой реализации оценить время работы нельзя. Всё зависит от временной сложности вырезки из коллекции.
Например, для простой реализации связного списка вырезка линейна, а значит сортировка станет кубической.
К чему это я, обобщать тип методов нужно с осторожностью.

Просто комментарий.